extensions/ql-vscode/src/view/model-editor/HiddenMethodsRow.tsx

```diff
@@ -1,35 +1,37 @@
-import {
-  VSCodeDataGridCell,
-  VSCodeDataGridRow,
-} from "@vscode/webview-ui-toolkit/react";
 import * as React from "react";
 import { styled } from "styled-components";
 import { pluralize } from "../../common/word";
+import { DataGridCell, DataGridRow } from "../common/DataGrid";
+import { ModelEditorViewState } from "../../model-editor/shared/view-state";
 
-const HiddenMethodsCell = styled(VSCodeDataGridCell)`
+const HiddenMethodsText = styled(DataGridCell)`
```

Comment:
Should these be renamed to `HiddenMethodsCell` again since we're now wrapping the cell again?

```suggestion
const HiddenMethodsCell = styled(DataGridCell)`
```